cmd/dagger/functions.go

```diff
@@ -123,27 +122,8 @@ func (fcs FuncCommands) All() []*cobra.Command {
 }
 
 func setCmdOutput(cmd *cobra.Command, vtx *progrock.VertexRecorder) {
-	if silent {
-		return
-	}
-
-	var stdout io.Writer
-	var stderr io.Writer
-
-	if stdoutIsTTY {
-		stdout = vtx.Stdout()
-	} else {
-		stdout = os.Stdout
-	}
-
-	if stderrIsTTY {
-		stderr = vtx.Stderr()
-	} else {
-		stderr = os.Stderr
-	}
-
-	cmd.SetOut(stdout)
-	cmd.SetErr(stderr)
+	cmd.SetOut(vtx.Stdout())
+	cmd.SetErr(vtx.Stderr())
```

Comment:
The frontend is now authoritative over `os.Stderr` and `os.Stdout` - nothing should ever directly print to them, instead it should create a Vertex with its ID as `idtui.PrimaryVertex` and write to `vtx.Stderr()` and `vtx.Stdout()`. Would like to make that smoother in the future, for now there's a bit of boilerplate.

The frontend handles buffering them so that we don't directly pipe to something like `grep` that will clobber the TUI output.

cc @helderco